doc/services/device_mgmt/ec_host_cmd.rst

```diff
@@ -36,13 +36,43 @@ one backend layer.
 .. image:: ec_host_cmd_shi.png
    :align: center
 
+Another case is SPI. Unfortunately, the current SPI API can't be used to handle the host commands
```

Comment:
@tbursztyka this seems like a problem which is causing big duplication, are there plans to address this?

Reply:
We are discussing the problem [here](https://github.com/zephyrproject-rtos/zephyr/issues/56091)